.github/workflows/premerge.yml

```diff
@@ -38,6 +38,7 @@ jobs:
         run: |
           python3 -m pip install --upgrade pip
           python3 -m pip install --no-cache-dir -e .[dev]
+          python3 -m pip install --no-cache-dir torch bitsandbytes
```

Comment:
seems to work for this new unit-test (line 24)

Reply:
while this would work, Chester suggest is better, as we use central place for these.

Ziyue could you add that dependency "bitsandbytes" in setup.cfg, after test_support: https://github.com/NVIDIA/NVFlare/blob/main/setup.cfg#L83

Reply:
@ZiyueXu77 actually, I see you have it in app_opt, do you want to put it in APP_OPT's dependency? https://github.com/NVIDIA/NVFlare/blob/main/setup.cfg#L57

Reply:
nice! updated